wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -167,9 +167,7 @@ sendMoney SendActions{..} passphrase moneySource dstDistr = do
                   fromMaybe (error "Corresponding adress meta not found")
                             (fst <$> find ((== addr) . snd) metasAndAdrresses)
           sk <- getSKByAddress passphrase addrMeta
-
-          ss <- withSafeSigner sk (pure passphrase) pure
-          pure $ fromMaybe (error "Coudln't get safe signer") ss
+          withSafeSignerUnsafe sk (pure passphrase) pure
```

Comment:
Yes, this is ok because we just derived this sk, and we previously did `checkPassMatches passphrase rootSk`.